### ISSUE TITLE
When adding due date, it sould appear as a date in calendar

### ISSUE THREAD

**Issue by David100mark** [nextcloud/deck#1560]
The deck app is very usefull for organisation and longterm projects. We use it a lot.
I really like the feature of due dates, and the possibility of getting notifications on my due dates, but i sometimes won't go on nextcloud for days and therefore forget about my tasks...

I would like to have all the tasks with due dates as calendar events. So everytime a new task with a due date is created a new event should be created aswell. If the task is only assigned to one (or multiple) person, the calendar event should only be visible in their calendar. 
When creating a new board, you can choose which calendar you want the events added to. 

I already considered adding them manualy, but it takes a lot of time and often, when delaying the due date, i forget to change it in my calendar. 

Edgy cases:
- If you get added to a board and you don't have a calendar yet, you should get asked if you want to create an new one.
- If you add a due date without a calendar assigned to the board, you should get a warning with the possibility to opt-out of future warnings.
- If the group using the board has a group calendar, task assigned to everyone should get added there

tl;dr:
add calendar event for due date of card

**Comment by juliushaertl** [comment]
This is a duplicate of #15, but here is some recent progress on this https://github.com/nextcloud/deck/pull/1545 :wink:
